### PR TITLE
Made initial User profile page

### PR DESCRIPTION
**Pull request by CoopTang**
- Profile page displays user information and edit link (#25)
- Profile page has link for user orders (#19)

### CODE REVIEW

File: app/views/partials/_user_info.html.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<p>Name: <%= name %></p>
```
Comment (lrs8810) [inline app/views/partials/_user_info.html.erb:1]: I really like this partial. Good work!

File: spec/features/users/profile_spec.rb [hunk 2]
```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+
+# As a registered user
```
Comment (lrs8810) [inline spec/features/users/profile_spec.rb:3]: Do we want to remove these in the future?

**Review by lrs8810 — APPROVED** [review]

File: app/views/partials/_user_info.html.erb (hunk 1 above)
Comment (freeheeling) [reply app/views/partials/_user_info.html.erb:1]: Agreed!

File: spec/features/users/navigation_spec.rb [hunk 3]
```diff
@@ -5,7 +5,15 @@
 RSpec.describe 'As a User' do
   describe 'as a Registered User' do
     it "has profile and logout links but doesn't have login or register links" do
-      user = User.create(name: 'Bob', address: '123 Main', city: 'Denver', state: 'CO', zip: 80_233, email: 'bob@email.com', password: 'secure')
+      user = User.create(
+        name: 'Bob', 
+        address: '123 Main', 
+        city: 'Denver', 
+        state: 'CO', 
+        zip: 80_233,
```
Comment (freeheeling) [inline spec/features/users/navigation_spec.rb:13]: Is zip stored as an integer or string?

**Review by freeheeling — COMMENTED** [review]
Great job, Michael!

File: spec/features/users/profile_spec.rb (hunk 2 above)
Comment (CoopTang) [reply spec/features/users/profile_spec.rb:3]: Yeah, I just forgot to remove these when I made the PR. Thanks for reminding me!

**Pull request merged**